vendor/go.etcd.io/etcd/client/v3/client.go

```diff
@@ -501,15 +502,15 @@ func (c *Client) checkVersion() (err error) {
 					return
 				}
 			}
-			if maj < 3 || (maj == 3 && min < 2) {
+			if maj < 3 || (maj == 3 && min < 4) {
```

Comment:
@serathius @logicalhan it's surprising to me the 3.5.x stream of etcd client would stop supporting 3.2 and 3.3 servers in a patch release. Is there a reference to why this was done? It seems like k8s needs a release note for this if updating this puts hard requirements on the etcd server version used. Also, where are supported minimum etcd versions for each k8s minor documented? Do we need to update that as well?

Reply:
This is an optional client feature `RejectOldCluster` that is not enabled on Kubernetes. Context https://github.com/etcd-io/etcd/issues/14323

Reply:
ah, ok